.github/workflows/ci.generate.ts

```diff
@@ -678,7 +683,7 @@ const ci = {
           run: [
             // output fs space before and after building
             "df -h",
-            "cargo build --locked --all-targets",
+            "cargo build -vv --locked --all-targets",
```

Comment:
```suggestion
            "cargo build --locked --all-targets",
```
not required anymore